src/rest-core.c

```diff
@@ -91,6 +92,18 @@ int rest_step(rest_context_t *rest, struct timeval *tv)
         request.http_verb = strdup("PUT");
         request.http_url = strdup(url);
         request.timeout = 20;
+        request.check_server_certificate = 0;
+        request.client_cert_file = o_strdup(rest->settings->http.security.certificate);
+        if(rest->settings->http.security.certificate != NULL && request.client_cert_file == NULL)
+        {
+            log_message(LOG_LEVEL_WARN, "[CALLBACK] Failed to set client certificate\n");
```

Comment:
If there's a failure in request configuration it should not be sent.

Reply:
Added abort sequence, please check if it is correct